.github/labels.yml

```diff
@@ -1,56 +1,31 @@
-- name: api changes
-  color: '5663e9'
-- name: api support
-  color: '5663e9'
 - name: apps:guide
   color: fbca04
 - name: apps:website
   color: fbca04
-- name: backlog
-  color: 7ef7ef
-- name: backport
-  color: 88aabb
 - name: blocked
+  description: Generic label to stop Kodiak from merging a pull request.
   color: fc1423
 - name: bug
-  color: d73a4a
-- name: caching
-  color: 80c042
-- name: chore
-  color: ffffff
-- name: ci
-  color: 0075ca
-- name: dependencies
-  color: 276bd1
+  description: Something is not working as expected.
+  color: '000000'
```

Comment:
Is this color change intentional?

Reply:
I feel black would stand out more as a bug. Welcome to feedback on this.

Reply:
I don't think i've actually seen black in the ui, so....what's it look like 🤷

Reply:
Absolutely not